asyncio_run_in_process/__init__.py

```diff
@@ -3,7 +3,7 @@
     InvalidState,
     ProcessKilled,
 )
-from .run_in_process import (  # noqa: F401
+from .main import (  # noqa: F401
```

Comment:
This rename was needed because the `run_in_process` function is imported here so it used to shadow the module with the same name, thus preventing me from monkey-patching stuff from the module in tests